app/views/catalogs/accordion/_providers.html.haml

```diff
@@ -28,8 +28,13 @@
 
     .panel.panel-default
       .panel-heading.provider-header{ class: [css_class(p), locked ? 'bg-danger text-danger' : ''], data: { id: p.id, process_ssr_found: provider_is_process_ssr.to_s, toggle: 'collapse', target: ".provider-#{p.id}", parent: "#institution-#{i.id}" } }
-        %h4.panel-title.text-center
-          = organization_name_display(p, locked)
+        .row
```

Comment:
From Manual Testing: The carets are showing up on providers, but not on institutions.

Reply:
Addressed